samples/Microsoft.Bot.Samples.Ai.Luis/Controllers/MessagesController.cs

```diff
@@ -2,52 +2,58 @@
 // Copyright(c) Microsoft Corporation.All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Bot.Builder;
-using Microsoft.Bot.Builder.Ai;
+using Microsoft.Bot.Builder.Adapters;
 using Microsoft.Bot.Builder.BotFramework;
-using Microsoft.Bot.Builder.Middleware;
+using Microsoft.Bot.Builder.Core.Extensions;
+using Microsoft.Bot.Builder.LUIS;
 using Microsoft.Bot.Schema;
+using Microsoft.Cognitive.LUIS;
 using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
 
-namespace Microsoft.Bot.Samples.Ai.Luis
+namespace Microsoft.Bot.Samples.Ai.Luis.Controllers
 {
     [Route("api/[controller]")]
     public class MessagesController : Controller
     {
         static BotFrameworkAdapter adapter;
-
-        /// <summary>
-        /// In this sample Bot, a new instance of the Bot is created by the controller 
-        /// on every incoming HTTP reques. The bot is constructed using the credentials
-        /// found in the config file. Note that no credentials are needed if testing
-        /// the bot locally using the emulator. 
-        /// </summary>        
+    
         public MessagesController(IConfiguration configuration)
         {
             if (adapter == null)
             {
-                adapter = new BotFrameworkAdapter(configuration)
-                    .Use(new LuisRecognizerMiddleware("xxxxxx", "xxxxxx"));
-
-                // LUIS with correct baseUri format example
-                //.Use(new LuisRecognizerMiddleware("xxxxxx", "xxxxxx", "https://xxxxxx.api.cognitive.microsoft.com/luis/v2.0/apps"))
-
+                adapter = new BotFrameworkAdapter(new ConfigurationCredentialProvider(configuration))
+                    .Use(new BatchOutputMiddleware())
```

Comment:
We shouldn't use this here, as it'll be gone in a day or two per feedback from folks. The in-person feedback at the Hack's is clear that folks prefers the "SendActivity" model where responses are just sent immediatly rather than batched.

Reply:
Great, should I wait for the removal of this middleware?